logx/log.go

```diff
@@ -171,14 +170,14 @@ func GetLogPushEntry(input *InputLog) LogPushEntry {
 	}
 
 	if input.HideToken {
-		token = hideToken(input.Token, 10)
+		input.Token = hideToken(input.Token, 10)
```

Comment:
I think you don't need to update `input.Token` here, we don't know where we will use this data. 
My suggestion: 

```suggestion
token := input.Token
if input.HideToken {
		token = hideToken(input.Token, 10)
}
```
And use `token` below

What do you think?

Reply:
Ok, cool. no problem from me